generator/resources/ecs_ec2_daemon_test_matrix.json

```diff
@@ -2,6 +2,13 @@
   {
     "instanceType":"t3a.xlarge",
     "ami": "cloudwatch-agent-integration-test-al2*",
-    "family": "linux"
+    "family": "linux",
+    "metadataEnabled": "enabled"
```

Comment:
I'm assuming targets means if it contains this then this can be a test case.

Reply:
Yes